html-render.js

```diff
@@ -57,56 +57,52 @@ const registerButtonSectionHelper = () => {
                       '<p>Modifying the button with "light" will ensure that the chosen button works on a black or darker-colored background, if applicable.</p>' +
                       '<h3>Light, Solid</h3><h4>Button Elements</h4>' +
                       '<div class="button-group">';
-    const headerH31 = '</div><h3>Button Elements</h3><div class="button-group">';
-    const headerH32 = '</div><h3>Anchor Elements</h3><div class="button-group">';
-    const headerH322 = '</div><h3>Anchor Elements</h3><div class="button-group" style="background-image:url(\'https://placeimg.com/767/300/nature\'); background-size:cover; background-position:top center;">';
-    const headerH33 = '</div><h3>Light, Solid</h3><h4>Button Elements</h4><div class="button-group">';
-    const headerH34 = '</div><h3>Light, Hollow</h3><h4>Button Elements</h4><div class="button-group">';
-    const headerH35 = '</div><h3>Light, Bar</h3><h4>Button Elements</h4><div class="button-group">';
-    const headerH36 = '</div><h3>Light, Opaque</h3><h4>Button Element</h4><div class="button-group" style="background-image:url(\'https://placeimg.com/767/300/nature\'); background-size:cover; background-position:top center;">';
-    const headerH37 = '</div><h3>Default</h3><div class="button-group">'
-    const headerH41 = '</div><h4>Button Elements</h4><div class="button-group">';
-    const headerH42 = '</div><h4>Anchor Elements</h4><div class="button-group">';
-    const headerH422 = '</div><h4>Anchor Element</h4><div class="button-group" style="background-image:url(\'https://placeimg.com/767/300/nature\'); background-size:cover; background-position:top center;">';
+    const headerH31 = '</div><h3>Anchor Elements</h3><div class="button-group">';
```

Comment:
There's probably a better way to name these variables to be more readable in the future if we ever return to this.